File: Assets/__Sample/Namespace.cs
using UnityEngine;
using socket.io;


namespace Sample {

    public class Namespace : MonoBehaviour {

        void Start() {
            var chat = Socket.Connect(Config.serverUrl + "/chat");
            var news = Socket.Connect(Config.serverUrl + "/news");

            chat.On("connect", () => {
                chat.Emit("hi");
            });
            chat.On("a message", (string r) => {
                Debug.Log(r);
            });

            news.On("connect", () => {
                chat.Emit("woot");
            });
            news.On("item", (string r) => {
                Debug.Log(r);
            });
        }

    }

}

File: Assets/__Sample/Server~/namespace.js
var app = require('http').createServer(handler)
var fs = require('fs');

app.listen(7001);

function handler (req, res) {
    fs.readFile(__dirname + '/index.html',
    function (err, data) {
        if (err) {
            res.writeHead(500);
            return res.end('Error loading index.html');
        }

        res.writeHead(200);
        res.end(data);
    });
}

// Start socket.io
var io = require('socket.io')(app);

// chat nsp
var chat = io
  .of('/chat')
  .on('connection', function (socket) {
    chat.emit('a message', { everyone: 'in', '/chat': 'will get' });
    socket.emit('a message', { that: 'only', '/chat': 'will get' });
  });

// news nsp
var news = io
  .of('/news')
  .on('connection', function (socket) {
    socket.emit('item', { news: 'item' });
  });